### ISSUE TITLE
Call of Duty: Advanced Warfare refusing to launch on ext4 under Windows

### ISSUE THREAD

**Issue by Skalt711** [bobranten/Ext4Fsd#24]
The game named in the title refuses to launch on an ext4 partition. The game seems to access files fine, however, it has some trouble reading it. It launches fine on an NTFS partition. Unfortunately, the game leaves no log whatsoever, so I can't tell where it fails. Curiously, a ""The IO Operation at logical block address for disk was retried" warning appears in the Windows event viewer each time upon launching fron an ext4 partition.

**Comment by bobranten** [comment]
I was interested in trying to debug this problem so I bought a used copy of the game for 10 dollar however when I try to install it from the DVD it says the installation will take 12 hours! Is that normal? Perhaps my laptop has some problem with the DVD player. I will try to do more tests if I find the time.

**Comment by Skalt711** [comment]
Yeah, it's strange that it takes 12 hours. DVD drives aren't that slow. Have you tried to copy the DVD content to hard drive and then launch the installation from there?

I can confirm that it is an Ext4fsd issue since the game launches fine from an Ext4 partition in Wine. My debugging skills suck, but multiple threads of the game exit with the NT_STATUS_CTL_FILE_NOT_SUPPORTED error that means "The network request is not supported" according to a common description, which might make sense since I had to install WinDivert to get Ext4fsd working, although I was hinted by vnepogodin (one of CachyOS developers) that it can mean a file attribute error. The issue appears both with 0.68 and 0.70b3.

**Comment by Skalt711** [comment]
I run Windows 11 build 22000, so WinDivert should be up to date.

**Comment by Skalt711** [comment]
Wait a minute, how did I get WinDivert in the first place? (Rhetoric question)

**Comment by Skalt711** [comment]
I meant WebClient (WebDAV), not WinDivert, sorry.

**Comment by Skalt711** [comment]
Also, ReShade can't read shaders no matter how I put the directory path in ReShade settings.
